checker/tests/nullness/Issue2215.java

```diff
@@ -0,0 +1,36 @@
+// Test Case for Issue2215
+// https://github.com/typetools/checker-framework/issues/2215
+// @skip-test until the bug is fixed
+
+import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class ParentClass {
+
+    @MonotonicNonNull Object parentClassField;
+
+    ParentClass(@Nullable Object o) {
+        this.parentClassField = o;
+    }
+}
+
+public class Issue2215 extends ParentClass {
+
+    @MonotonicNonNull Object f;
+    @MonotonicNonNull Object g = null;
+    @MonotonicNonNull Object h = new Object();
+
+    Issue2215(@Nullable Object o) {
+
+        super(o);
+        this.parentClassField = o;
+        this.f = o;
+        this.f = null;
```

Comment:
I think this should be an error.  It might set a field from a non-null value to a null value.
The issue in the issue tracker already gets this case right.
I think we should close this pull request, because we have spent a lot of time iterating, mostly just trying to get to what is already in the example in the pull request.  (I just added the supertype to it.)

Reply:
> I think this should be an error. It might set a field from a non-null value to a null value.
> The issue in the issue tracker already gets this case right.

I agree with you.

Thank you very much for correcting the Example in the Issue tracker.